dali/test/python/test_RN50_data_fw_iterators.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import glob
+import argparse
+import time
+import tensorflow as tf
+
+data_paths = ["/data/imagenet/train-jpeg"]
```

Comment:
Maybe running on `val-jpeg` would be enough for this test. In the interest of time and CI resources.

Reply:
there is an option to run specific number of iterations during L0, so I think it's fine. In L3 we would still go through the whole imagenet